torchvision/models/quantization/googlenet.py

```diff
@@ -70,48 +78,48 @@ def googlenet(pretrained=False, progress=True, quantize=False, **kwargs):
 
         if not original_aux_logits:
             model.aux_logits = False
-            model.aux1 = None
-            model.aux2 = None
+            model.aux1 = None  # type: ignore
```

Comment:
Let me know if there is any alternative.

Reply:
regarding mypy error ignore, it's much better if you specify the error type that you want to avoid!
So say I get a `return-type` error then `# type: ignore[return-type]`

This way if mypy throws a new error after some modifications, we know that it wasn't supposed to do this

Reply:
Nice! I didn't know this. I will change the code